src/main/java/com/spartronics4915/frc2020/commands/RaisePanelRotatorCommand.java

```diff
@@ -0,0 +1,39 @@
+package com.spartronics4915.frc2020.commands;
```

Comment:
Please change your code to use four spaces for indentation instead of two, and run our formatter.